samples/VSAndroidApp/Resources/AboutResources.txt

```diff
@@ -0,0 +1,44 @@
+Images, layout descriptions, binary blobs and string dictionaries can be included 
```

Comment:
I think we can exclude this file and the `AboutAssets.txt` file to reduce the additions a bit.

Reply:
I would like to keep it just clean template app, without any changes, so I think it is ok to include these.